src/native/chunk/splitter.cpp

```diff
@@ -86,7 +87,10 @@ Splitter::finish(uint8_t* md5, uint8_t* sha256)
     if (md5) {
         if (_md5_mb_ctx) {
             md5_mb_submit_and_flush(0, 0, HASH_LAST);
-            memcpy(md5, hash_ctx_digest(_md5_mb_ctx), MD5_DIGEST_NWORDS * 4);
+            uint32_t *digest = reinterpret_cast<uint32_t*>(md5);
+            for (int i = 0; i < MD5_DIGEST_NWORDS; i++) {
+                digest[i] = le32toh(hash_ctx_digest(_md5_mb_ctx)[i]);
+            }
```

Comment:
@liranmauda just to make sure I understand - this changed between our previous isa-l_crypto version and the new one so that now md5 words are LE but before it was BE ?? It just sounds weird that this changed between versions no? What am I missing?

Reply:
It seems that it was changed between isa-l_crypto versions. 
@jackyalbo even open a question there: https://github.com/intel/isa-l_crypto/issues/57

Reply:
ok then